incubator/elasticsearch/templates/configmap.yaml

```diff
@@ -93,60 +93,35 @@ data:
 {{- end }}
   pre-stop-hook.sh: |-
     #!/bin/bash
-    set -e
```

Comment:
Sorry if I was not clear.
I kept the `set -e` for the `pre-stop-hook` (I removed from the `post-stop-hook`)

Reply:
@yanns  `set -e` does not matter on `pre-stop-hook.sh` too. So I remove it and make a code as short as possible.